util/span: TestBTreeDeleteInsertCloneEachTime failed

util/span.TestBTreeDeleteInsertCloneEachTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12985149?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/12985149?buildTab=artifacts#/) on master @ [3771f53b7d9d4d480d588647b91bc0988bfbc056](https://github.com/cockroachdb/cockroach/commits/3771f53b7d9d4d480d588647b91bc0988bfbc056):

Fatal error:

```
panic: test timed out after 57s
running tests:
	TestBTreeDeleteInsertCloneEachTime (10s)
	TestBTreeDeleteInsertCloneEachTime/reset=false (10s)

```
Stack: 

```
goroutine 55 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x320
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBTreeDeleteInsertCloneEachTime
=== RUN   TestBTreeDeleteInsertCloneEachTime/reset=false
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/unowned
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBTreeDeleteInsertCloneEachTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>


Jira issue: CRDB-34119

util/span.TestBTreeDeleteInsertCloneEachTime [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/13097706?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/13097706?buildTab=artifacts#/) on master @ [1807f8b4f7cea702ce5f4744bbc2767fed994023](https://github.com/cockroachdb/cockroach/commits/1807f8b4f7cea702ce5f4744bbc2767fed994023):

Fatal error:

```
panic: test timed out after 57s
running tests:
	TestBTreeDeleteInsertCloneEachTime (37s)
	TestBTreeDeleteInsertCloneEachTime/reset=true (18s)

```
Stack: 

```
goroutine 63 [running]:
testing.(*M).startAlarm.func1()
	GOROOT/src/testing/testing.go:2259 +0x320
created by time.goFunc
	GOROOT/src/time/sleep.go:176 +0x38
```
<details><summary>Log preceding fatal error</summary>
<p>

```
=== RUN   TestBTreeDeleteInsertCloneEachTime
=== RUN   TestBTreeDeleteInsertCloneEachTime/reset=false
=== RUN   TestBTreeDeleteInsertCloneEachTime/reset=true
```
</p>
</details>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestBTreeDeleteInsertCloneEachTime.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/util/span/BUILD.bazel
+++ b/pkg/util/span/BUILD.bazel
@@ -24,7 +24,7 @@ go_library(
 
 go_test(
     name = "span_test",
-    size = "small",
+    size = "large",
     srcs = [
         "frontier_test.go",
         "frontierentry_interval_btree_test.go",  #keep

